binjr-core/src/main/java/eu/binjr/core/dialogs/DataAdapterDialog.java

```diff
@@ -118,7 +118,11 @@ public DataAdapterDialog(Node owner, Mode mode, String mostRecentListName, boole
                 updateUriAutoCompletionBinding();
             }
         });
-        uriField.getSelectionModel().selectedItemProperty().addListener((obs, oldText, uri) -> validateUri(uri));
+        uriField.getSelectionModel().selectedItemProperty().addListener((obs, oldText, uri) -> {
+            if (!uri.equals("")) {
```

Comment:
Please consider using `!uri.isEmpty()` instead, as it is the canonical way to check for empty strings.
Alternatively, you may want to consider using ìsBlank()` instead, which returns true if the string is empty or contains only white space codepoints, otherwise false.
I'm honestly not sure it makes sense in this case, so I'll leave it up to you whether to use ìsEmpty` or `isBlank`